src/express-api/server.ts

```diff
@@ -23,23 +21,22 @@ app.use(cors({
   origin: allowedOrigin
 }));
 
-const fileSizeLimit = process.env.IPFS_MAX_FILE_SIZE
-
+const fileSizeLimit = process.env.IPFS_MAX_FILE_SIZE_BYTES || "2097152" // 2 MB in bytes as string
```

Comment:
`IPFS_MAX_FILE_SIZE` in `localhost.env` but `IPFS_MAX_FILE_SIZE_BYTES` in `server.ts`